modules/rte_mis_school/rte_mis_school.module

```diff
@@ -648,12 +650,31 @@ function rte_mis_school_udise_list(FieldStorageConfig $definition, ContentEntity
     'field_workflow' => 'school_workflow_approved',
   ]);
 
+  // Perform an entity query to get user IDs
+  // with roles 'school' and 'school_admin'.
+  $query = \Drupal::entityQuery('user')
+    ->condition('roles', ['school', 'school_admin'], 'IN')
+    ->accessCheck(TRUE);
+  $user_ids = $query->execute();
+
+  $usernames = [];
+  if (!empty($user_ids)) {
+    // Load only the usernames of these users.
+    $users = User::loadMultiple($user_ids);
+    foreach ($users as $user) {
+      $usernames[] = $user->getAccountName();
+    }
+  }
+
   $options = [
     '_none' => '- None -',
   ];
   foreach ($terms as $term) {
     if ($term instanceof Term) {
-      $options[$term->id()] = $term->label();
+      // Don't load the schools whose account is created.
+      if (!in_array($term->label(), $usernames)) {
+        $options[$term->id()] = $term->label();
+      }
```

Comment:
Load all the users first of school, school_admin, and then load the school taxonomy, not including that username.

Reply:
sure